### PR TITLE
Clarify disabling Command Log

### PR DESCRIPTION
**Pull request by j14t**

### CODE REVIEW

File: content/guides/references/troubleshooting.md [hunk 1]
```diff
@@ -443,6 +443,13 @@ environment variable below during `cypress open` or `cypress run`.
 CYPRESS_NO_COMMAND_LOG=1 cypress run
 ```
 
+Disabling the Command Log can make tests faster as there is no time spent
+displaying the command log and updating it as tests run. The downside is,
+when there is a failure you cannot see what command failed.
+
+**Note:** In headless mode screenshots and videos will not include the Command
```
Comment (marktnoonan) [inline content/guides/references/troubleshooting.md:450]: > In headless mode screenshots and videos will not include the Command Log.

Really it's in any mode: open , run headed, or run headless. No command log will render when a spec is running.


```suggestion
**Note:** With this variable set, screenshots and videos will not include the Command Log
```

File: content/guides/references/troubleshooting.md [hunk 2]
```diff
@@ -443,6 +443,13 @@ environment variable below during `cypress open` or `cypress run`.
 CYPRESS_NO_COMMAND_LOG=1 cypress run
 ```
 
+Disabling the Command Log can make tests faster as there is no time spent
+displaying the command log and updating it as tests run. The downside is,
+when there is a failure you cannot see what command failed.
```
Comment (marktnoonan) [inline content/guides/references/troubleshooting.md:448]:
```suggestion
With this variable set, Cypress will skip rendering the Command Log entirely, and perform none of the usual DOM updates to display information about commands and statuses as the test runs. 
```

**Review by marktnoonan — CHANGES_REQUESTED** [review]

**Review by marktnoonan — APPROVED** [review]

**Comment by marktnoonan** [comment]
Thanks @j14t, I approved but somebody from @cypress-io/dx should get eyes on this also before merge.

**Review by jaffrepaul — APPROVED** [review]
LGTM 🎉

**Comment by j14t** [comment]
Can someone merge? I don't have permissions.

**Comment by admah** [comment]
@j14t sorry this has taken a bit to get in. Could you see what the `lint` failure is?

**Pull request merged**